content/docs/futures/combinators.md

```diff
@@ -351,7 +351,7 @@ use futures::{future, Future};
 use std::io;
 use std::sync::Arc;
 
-fn get_motd() -> impl Future<Item = String, Error = io::Error> {
+fn get_message() -> impl Future<Item = String, Error = io::Error> {
```

Comment:
I was unaware of what `motd` meant and I imagine others will be the same. I think changing to `message` removes an unnecessary distraction.